main/src/sphexa/CMakeLists.txt

```diff
@@ -34,6 +58,7 @@ endif()
 if(CMAKE_CUDA_COMPILER)
     add_executable(${exename}-cuda $<TARGET_OBJECTS:gather_obj> $<TARGET_OBJECTS:cuda_find_neighbors_obj> $<TARGET_OBJECTS:cuda_sph> sphexa.cpp)
     target_include_directories(${exename}-cuda PRIVATE ${SPH_EXA_INCLUDE_DIRS})
+    target_include_directories(${exename}-cuda PUBLIC ${CMAKE_BINARY_DIR}/main/src/util)
```

Comment:
I would choose `main/src/` as the location here to be consistent with the source tree